WorkspaceServer.Tests/Kernel/XplotKernelExtensionTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+using System.Linq;
+using System.Threading.Tasks;
+using WorkspaceServer.Kernel;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace WorkspaceServer.Tests.Kernel
+{
+    public class XplotKernelExtensionTests : CSharpKernelTestBase
+    {
+        public XplotKernelExtensionTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task When_a_chart_is_returned_the_value_produced_has_html_with_the_require_config_call()
+        {
+            var kernel = CreateKernel();
+            kernel.UseDefaultExtensions();
+
+            await kernel.SendAsync(new SubmitCode("#r nuget:XPlot.Plotly"));
+            await kernel.SendAsync(new SubmitCode("using XPlot.Plotly;"));
+            await kernel.SendAsync(new SubmitCode("new PlotlyChart()"));
+
+            KernelEvents
+                .ValuesOnly()
+                .OfType<ValueProduced>()
+                .Should().
+                ContainSingle(valueProduced =>
+                    valueProduced.FormattedValues.Any(formattedValue =>
+                       formattedValue.Value.ToString().Contains("require.config({paths:{plotly:\'https://cdn.plot.ly/plotly-latest.min\'}});")
+                 ));
+        }
+
+        [Fact]
+        public async Task When_a_chart_is_returned_the_value_produced_hash_the_plotly_newPlot_call()
+        {
+            var kernel = CreateKernel();
+            kernel.UseDefaultExtensions();
+
+            await kernel.SendAsync(new SubmitCode("#r nuget:XPlot.Plotly"));
```

Comment:
I think for now this shouldn't be needed (although we'll need to modify the ScriptOptions in `CSharpKernel`). Like `PocketView` and the display helper, it should be available by default.

Reply:
In that case things will work in c# out of the box and not for fsharp unless we explicitly add that dependency for f# as well. Do we need a list of references "somewhere" and those should be added to both f# and c# by default ?

Reply:
@brettfo Thoughts?